templates/cpp-template-default/proj.android-studio/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.3.0'
+        classpath 'com.android.tools.build:gradle:2.2.0-beta3'
```

Comment:
Done, but there's a resource conflict now. See http://b.android.com/223083. Do you actually need both the `.pvr` and the `.pvr.gz` file of the same name?